extensions/synonyms/README.md

```diff
@@ -0,0 +1,3 @@
+# Synonyms
+
+Find synonyms for words and translate if necessary
```

Comment:
I think we need to explain more about what it does and which services it uses :)

We have other alternatives in the store. Why use this one?

Reply:
I added a more in depth description, thank you

I already use this extension every day and though I should publish it, too useful